tests/test_targets.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2021 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import absolute_import
+
+import pytest
+
+from pex.interpreter import PythonInterpreter
+from pex.orderedset import OrderedSet
+from pex.platforms import Platform
+from pex.targets import AbbreviatedPlatform, LocalInterpreter, Target, Targets
+
+
+@pytest.fixture
+def current_interpreter():
+    # type: () -> PythonInterpreter
+    return PythonInterpreter.get()
+
+
+def test_interpreter(
```

Comment:
These tests were mainly just moved from `tests/test_distribution_targets`. The mutex tests of the 2-state `DistributionTarget` type were obsoleted and killed.